modular_skyrat/modules/organs/code/brain.dm

```diff
@@ -0,0 +1,120 @@
+//Unifying vox brains and android brains, so that when one is updated the other is as well.
+//Vox and basic cybernetic (cortical) brains will have both the robotic and organic brain surgery options; while android only has robotic.
+
+//Generic cyber brain parent
+/obj/item/organ/internal/brain/cybernetic
+	name = "generic cybernetic brain"
+	desc = "this is a parent for the other brains. spawn those"
+	icon = 'modular_skyrat/master_files/icons/obj/medical/organs/organs.dmi'
+	icon_state = "brain-a"
+	organ_flags = ORGAN_ROBOTIC | ORGAN_VITAL
+	
+/obj/item/organ/internal/brain/cybernetic/emp_act(severity)
+	. = ..()
+	if(. & EMP_PROTECT_SELF)
+		return
+	if(owner.stat == DEAD)
+		return
+	switch(severity)
+		if(1)
+			to_chat(owner, span_boldwarning("You feel [pick("like your brain is being fried", "a sharp pain in your head")]!"))
+			owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, 20, 150)
+			owner.set_jitter_if_lower(30 SECONDS)
+			owner.adjust_stutter(30 SECONDS)
+			owner.adjust_confusion(10 SECONDS)
+		if(2)
+			to_chat(owner, span_warning("You feel [pick("disoriented", "confused", "dizzy")]."))
+			owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, 10, 150)
+			owner.set_jitter_if_lower(30 SECONDS)
+			owner.adjust_stutter(30 SECONDS)
+			owner.adjust_confusion(3 SECONDS)
```

Comment:
This will still get affected by the parent running
```
/obj/item/organ/internal/brain/cybernetic/emp_act(severity)
	. = ..()
	if(. & EMP_PROTECT_SELF)
		return
	switch(severity) // Hard cap on brain damage from EMP
		if (EMP_HEAVY)
			apply_organ_damage(20, BRAIN_DAMAGE_SEVERE)
		if (EMP_LIGHT)
			apply_organ_damage(10, BRAIN_DAMAGE_MILD)
```
hence why it was it's own type

Reply:
Oh right I need to go blank that out, I did it for the vox alr